WoWPro/WoWPro_ZoneData.lua

```diff
@@ -503,7 +503,7 @@ DefineZone9(0420, "Ruins of Ogudei", UIMapType.Micro, 0418, 213)
 DefineZone9(0421, "Reliquary Incursion@Ruins of Ogudei", UIMapType.Micro, 0418, 213)
 DefineZone9(0422, "Dread Wastes", UIMapType.Zone, 0424, nil, 0451, 0474, 0475)
 DefineZone9(0423, "Silvershard Mines!Dungeon", UIMapType.Dungeon, 0224, nil)
-DefineZone9(0424, "Pandaria", UIMapType.Continent, 0947, nil, 0371, 0376, 0379, 0388, 0390, 0418, 0422, 0433, 0504, 0507, 0516, 0554, 1502)
```

Comment:
This looks wrong 0390 and 1530 are both "Vale of Eternal Blossoms" and should both be in this list.